ui/templates/header.html

```diff
@@ -12,7 +12,7 @@
             Sign Out
           </a>
           {% else %}
-          <a class="members-login btn ghost upper-case" href="{% url 'social:begin' 'edxorg' %}">Log In</a>
+          <a class="members-login btn ghost open-signup-dialog upper-case">Log In</a>
           <a class="btn btn-login white-button open-signup-dialog upper-case">
```

Comment:
This is probably outside of the scope of this button request, but for accessibility purposes, both the "Log In" and "Sign Up" buttons should use the `<button>` tag, not the `<a>` tag. The `<button>` tag is used to indicate some action that the user can take on the page, which does not immediately bring them to another page. The `<a>` tag is used to indicate a link to a different page on the internet. Since both "Log In" and "Sign Up" open a dialog rather than taking the user to another page, they should be `<button>`s.